packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
```

Comment:
Do not vary the component ID based on RTL. Please see the explanation of "Components" under https://competent-shirley-cc2f82.netlify.com/theming/#default_theme. The consumer has access to props in order to make styling tweaks. This flavor of ID is unnecessary (and unexpected based on all of our other styled components).

Reply:
Thanks for the suggestion 👍. I have removed component ID related code in this styled component because it was unnecessary and unexpected.